content/docs/guides/flyway-multiple-environments.md

```diff
@@ -0,0 +1,211 @@
+- A database. This guide uses the ready-to-use `neondb` database on the `main` branch of your Neon project. You can create your own database if you like. See [Create a database](/docs/manage/databases#create-a-database) for instructions.
+
+## Add a table to your database
+
+Set up a database to work with by adding a table to your `neondb` database on the `main` branch of your Neon project. We'll consider this your _production_ environment database.
+
+Open the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor), and run the following statement:
+
+```bash
+create table person (
+    ID int not null,
+    NAME varchar(100) not null
+);
+```
+
+## Create databases for development and staging
+
+Using Neon's branching feature, create your _development_ and _staging_ databases. When you create a branch in Neon, you are creating a copy-on-write clone of the parent branch that incudes all databases and roles that exist on the parent, and each branch is an isolated Postgres instance with its own compute resources.
+
+Perform these steps twice, once for your _development_ branch and once for your _staging_ branch.
+
+1. In the Neon Console, select your project.
+2. Select **Branches**.
+3. Click **New Branch** to open the branch creation dialog.
+4. Enter a name for the branch. For example, name the branch for the environment (_development_ or _staging_).
+5. Select a parent branch. This should be the branch where you created the `person` table.
+6. Leave the other default settings and click **Create Branch**.
+
+When you are finished, you should have a _development_ branch and a _staging_ branch.
+
+Alternatively, you can create your _development_ and _staging_ branches using the [Neon API](https://api-docs.neon.tech/reference/createprojectbranch) or [Neon CLI](/docs/reference/cli-branches#create).
+
+<CodeTabs labels={["API", "CLI"]}>
+
+```bash
+curl --request POST \
+     --url https://console.neon.tech/api/v2/projects/project_id/branches \
+     --header 'accept: application/json' \
+     --header 'authorization: Bearer $NEON_API' \
+     --header 'content-type: application/json' \
+     --data '
+{
+  "branch": {
+    "name": "development"
+  },
+  "endpoints": [
+    {
+      "type": "read_only"
+    }
+  ]
+}
+' | jq
+```
+
+```bash
+neonctl branches create --name development
+```
+
+</CodeTabs>
+
+## Retrieve your Neon database connection strings
+
+From the Neon **Dashboard**, retrieve the connection string for each branch (`main`, `development`, and `staging`) from the **Connection Details** widget. Use the **Branch** drop-down menu to select each branch before copying the connection string.
+
+Your connection strings should look something like the ones shown below. Note that the hostname differs for each (the part starting with `ep-` and ending with `aws.neon.tech`). That's because each branch is hosted on its own compute instance.
+
+- **main**
```

Comment:
We don't want to overuse tabs, but you could consider it here.

It might be a nice touch to be able to highlight relevant portions of a string.

Or mud at the wall, color code highlighting to include a legend beneath to identify the parts of the string?

Reply:
I'll leave this for now, since we mention that connection strings "look something like" these. We can look at a possible convention to use at some point.